vms/shift/views.py

```diff
@@ -82,23 +79,23 @@ def add_hours_manager(request, shift_id, volunteer_id):
                 end_time = form.cleaned_data['end_time']
                 try:
                     if(end_time>start_time):
-                        	 add_shift_hours(
-                           	 volunteer_id,
-                           	 shift_id,
-                           	 start_time,
-                           	 end_time
-                            	 )
-                        	 return HttpResponseRedirect(reverse(
-                        	 'shift:manage_volunteer_shifts',
-                        	 args=(volunteer_id, )
-                        	 ))
-		    else:
-				messages.add_message(request, messages.INFO, 'End time should be greater than start time')
-				return render(
-                    		request,
-                    		'shift/add_hours_manager.html',
-                    		{'form': form, 'shift_id': shift_id, 'volunteer_id': volunteer_id, }
-                    		)
+                        add_shift_hours(
+                            volunteer_id,
+                            shift_id,
+                            start_time,
+                            end_time
+                            )
+                        return HttpResponseRedirect(reverse(
+                            'shift:manage_volunteer_shifts',
+                            args=(volunteer_id, )
+                            ))
+                    else:
+                        messages.add_message(request, messages.INFO, 'End time should be greater than start time')
+                        return render(
+                            request,
+                            'shift/add_hours_manager.html',
+                            {'form': form, 'shift_id': shift_id, 'volunteer_id': volunteer_id, }

```

Comment:
@ Yes, in the second [ commit](https://github.com/smarshy/vms/commit/16fb2ead0eb4d05ff208a67dd88f6800451d323a)  I have just fixed the indentation for edit hours, add hours, add hours manager, edit hours manager. The code seemed to work fine and no issues were reported.